src/egecontrolbase.cpp

```
@@ -11,6 +11,10 @@ egeControlBase基类定义和相关操作
 #define _ALLOW_RUNTIME_LIBRARY_MISMATCH
 #endif
 
+#ifndef _CRT_SECURE_NO_WARNINGS
+#define _CRT_SECURE_NO_WARNINGS
+#endif
```

Cppcheck (2.10-2)

[performance] Variable 'm_target' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)